aspnetcore/mvc/views/view-compilation.md

```diff
@@ -16,6 +16,22 @@ uid: mvc/views/view-compilation
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-Razor views are compiled at runtime when the view is invoked. ASP.NET Core 1.1.0 and higher can optionally compile Razor views and deploy them with the app (precompilation). To deploy precompiled views, set `MvcRazorCompileOnPublish` to true and include a package reference to `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation`. The following *.csproj* sample highlights these settings:
+Razor views are compiled at runtime when the view is invoked. ASP.NET Core 1.1.0 and higher can optionally compile Razor views and deploy them with the app (precompilation). 
 
-[!code-html[Main](view-compilation\sample\MvcRazorCompileOnPublish.csproj?highlight=5,12)]
+To deploy precompiled views:
+
+# [ASP.NET Core 1.x](#tab/aspnet1x)
+
+Set `MvcRazorCompileOnPublish` to true and include a package reference to `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation`. The following *.csproj* sample highlights these settings:
+
+[!code-xml[Main](view-compilation\sample\MvcRazorCompileOnPublish.csproj?highlight=5,12)]
+
+# [ASP.NET 2.0](#tab/aspnet20)
+
+Set `MvcRazorCompileOnPublish` to true. The following *.csproj* sample highlights this setting:
+
+[!code-xml[Main](view-compilation\sample\MvcRazorCompileOnPublish2.csproj?highlight=5)]
```

Comment:
We should either set version to 2.0.0 in all these .csproj examples or warn in text, and we should have an issue for getting them set to 2.0.0 as soon as possible after it's released.